config/envs/rhpam-authoring-ha.yaml

```diff
@@ -228,6 +529,23 @@ others:
           annotations:
             description: The broker's OpenWire port.
 
+      - spec:
+          clusterIP: None
+          ports:
+            - port: 8888
+              protocol: TCP
+              targetPort: 8888
+          selector:
+            deploymentConfig: "[[.ApplicationName]]-amq"
+        metadata:
+          name: "[[.ApplicationName]]-amq-ping"
```

Comment:
I suppose that this is AMQ scaledown controller ping service. The name of this service needs to be "ping" (as used in template https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.3.x/templates/rhpam73-authoring-ha.yaml#L770). The reason is that ping service for AMQ scaledown controller is hardcoded, see AMQ scaledown controller

Reply:
@sutaakar 
i set this env var to resolve that issue and allow for custom service names:
https://github.com/kiegroup/kie-cloud-operator/blob/f308ce8d29101824a90b75ed20d2b7d797a144d9/config/envs/rhpam-authoring-ha.yaml#L452-L453

i get the following in logs and amq runs fine:
```shell
2019-03-29 13:58:26,630 INFO  [org.jgroups.protocols.openshift.DNS_PING] serviceName [rhpam-authoring-ha-amq-ping] set; clustering enabled
```